backend/app/models/inquiry.py

```diff
@@ -1,27 +1,33 @@
 import uuid
 from datetime import datetime, timezone
+from typing import List, Optional
 
-from sqlmodel import Field, SQLModel
+from sqlmodel import Field, Relationship, SQLModel
+
+
+class Response(SQLModel, table=True):
+    id: uuid.UUID = Field(default_factory=uuid.uuid4, primary_key=True)
+    content: str = Field(min_length=1, max_length=255)
+    inquiry_id: uuid.UUID = Field(foreign_key="inquiry.id")
+    inquiry: Optional["Inquiry"] = Relationship(back_populates="responses")
```

Comment:
Mostly my ignorance here -- what's the purpose of this?

Reply:
the purpose of the Response class? or the inquiry field?

Reply:
I created a bidirectional relationship between Response and Inquiry models. 
This way; from a Response instance access Inquiry instance, and vice versa.

Reply:
What's the scenario where we need to access inquiry from response?

Reply:
possible validation if the response was correctly matched to the inquiry, tracking, debugging